plugins/@grouparoo/app-templates/src/source/table/templates.ts

```diff
@@ -239,6 +239,12 @@ async function loadTablesAndColumns(
           });
         }
       }
+
+      if (params.with.replace(/['"]+/g, "") && map.length === 0) {
+        throw new Error(
+          'Could not find any listed columns in source. If using * make sure you used quotes ("*")?'
+        );
+      }
```

Comment:
Note: This change helps avoid an error in bash/zsh in which an un-quoted `*` (vs `"*"`) would be expanded to match the files in this directory, and not be the string `"*"` like we wanted.

Reply:
This came in via #1502. Rebasing removed the change.